app/assets/javascripts/discourse/app/templates/user/messages.hbs

```diff
@@ -105,26 +111,29 @@
         {{/link-to}}
       </li>
     {{/if}}
-
-    {{#unless mobileView}}
-      {{#if group}}
-        <li>
-          {{group-notifications-button
-            value=group.group_user.notification_level
-            onChange=(action "changeGroupNotificationLevel")
-          }}
-        </li>
-      {{/if}}
-
-      {{#if showNewPM}}
-        <li>
-          {{d-button class="btn-primary new-private-message" action=(route-action "composePrivateMessage") icon="envelope" label="user.new_private_message"}}
-        </li>
-      {{/if}}
-    {{/unless}}
   {{/mobile-nav}}
 {{/d-section}}
 
+{{#if site.mobileView}}
```

Comment:
Minor but another place where we could use one `{{#if (and site.mobileView showNewPm)}}`